@here/harp-mapview-decoder/test/TileLoaderTest.ts

```diff
@@ -201,9 +201,9 @@ describe("TileLoader", function() {
 
                 // tslint:disable-next-line: no-shadowed-variable
                 const loadPromise = tileLoader.loadAndDecode();
-                // tslint:disable-next-line: no-unused-expression
                 expect(loadPromise).to.not.be.undefined;
 
+                LoggerManager.instance.update("TileLoader", { enabled: true });
```

Comment:
I've specifically proposed to use `before` and `after` hooks, because now if test fail between "enable"/"disable" lines, loggger will remain disabled for rest of test execution.

While it is minor, test cases should leave environment unchanged even if it's only logger configuration.

... or, we can use async/await and good old `try catch` although we may discuss if it's elegant.

Reply:
good point, I missed that, sorry. Will fix.

Reply:
question, if I use the before / after hooks, the logger is disabled anyway for all tests, so it provides less cover.

I will use finally, that seems to be the best usage.